app/build.gradle

```diff
@@ -4,9 +4,9 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'de.timfreiheit.resourceplaceholders'
 
-def keystorePropertiesFile = rootProject.file("keystore.properties")
```

Comment:
even if you did these changes locally, you definitely shouldnt have commited them. Revert it please.

Reply:
This file is not in the master branch. In order to build the code, I'm assuming everyone has to comment these lines.

Reply:
you shouldnt care about everyone else, only about the issue you are fixing right at the moment

Reply:
- Everyone includes me. How would I build the code successfully without the "keystore.properties" file?

Reply:
you can change any code locally, but you shouldnt push it to the master branch